draft-ietf-rtcweb-jsep.xml

```diff
@@ -2765,8 +2796,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                   "sendonly", prepare to start transmitting media using the specified primary SSRC
                   and one of the selected payload types, once the underlying transport
                   layers have been established. Otherwise, stop transmitting RTP media,
-                  although RTCP should still be sent, as described in
-                  <xref target="RFC3264"/>, Section 5.1. </t>
+                  although RTCP should still be sent, as described in 
+                  <xref target="RFC3264"/>, Section 5.1. If RID values are specified, include the RID header
+		  extension in the RTP streams, as indicated in <xref
+		  target="I-D.ietf-mmusic-rid"></xref>, Section 4).  If simulcast is negotiated,
+		  send the number of Source RTP Streams as specified in
+		  <xref target="I-D.ietf-mmusic-sdp-simulcast"></xref>, Section 6.2.2.
+		  The "send" portion of any "a=simulcast" line from a
+		  remote description MUST be ignored, since receiving
+		  of simulcast is not supported.
```

Comment:
I originally wrote it like that and Justin's response was that we can leave that for a new draft/spec when we need it.

I do agree with him that the way I wrote it before, leaving that open, was more complex.